### PR TITLE
e2e: password change

### PR DESCRIPTION
**Pull request by remon-nashid**

### CODE REVIEW

**Comment by remon-nashid** [comment]
@wahidrahim let me know if I missed any cases.

File: __tests__/utils.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+import puppeteer from 'puppeteer';
+
+export async function refillTextInput (handle: puppeteer.ElementHandle<Element>, text: string): Promise<puppeteer.ElementHandle<Element>> {
```
Comment (wahidrahim) [inline __tests__/utils.ts:4]: Wondering what this is about exactly; some kind of necessary hack to make puppeteer play nice?

**Review by wahidrahim — APPROVED** [review]
`Change password` spec looks good 👍 👍 

It covers everything relevant. The only cases I can think of are probably redundant:

- "Validates required fields": basically leaving everything empty and trying to submit
- "Validates current password length": I noticed current password can also show a "Password too short" error

Approved!

File: __tests__/utils.ts (hunk 1 above)
Comment (remon-nashid) [reply __tests__/utils.ts:4]: We need to clear input value from test to test, but Puppeteer doesn't provide direct access to DOM. Clearing input value has to be done manually, ie triple-click to select and press back button. This utility just saves typing.

**Pull request merged**